### PR TITLE
Feature/thongNguyen/SwiftOOP

### PR DESCRIPTION
**Pull request by mvn-thongnguyen-dn**

### CODE REVIEW

File: 03_SwiftOOP/ThongNT/BaiTap1-4.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+//----------------------------------//
+// Bài tập 3 //
+// class Tam giác thừa kế từ đa giác
+final class TamGiac: DaGiac {
+    var chuVi: Int = 0
+    var values: [Int] = []
+    
+    init?(cacCanh: [Int]) {
+        super.init(soCanh: 3, kichThuoc: cacCanh)
```
Comment (mvn-bienle-dn) [inline 03_SwiftOOP/ThongNT/BaiTap1-4.playground/Contents.swift:129]: Need check case:
- mangKichThuoc.count < 3 or > 3  on init function
- Self is a triangle

**Review by mvn-bienle-dn — COMMENTED** [review]
@mvn-thongnguyen-dn Please check my comment. Thank you!

**Comment by mvn-bienle-dn** [comment]
Resolve conflict please.

File: 03_SwiftOOP/ThongNT/BaiTap5-10.playground/Contents.swift [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+    
+    init(name: String, yearOfBirth: Int, totalGrade: Float) {
+        self.name = name
+        self.yearOfBirth = yearOfBirth
+        self.totalGrade = totalGrade
+    }
+}
+
+final class Classroom {
+    var studentList : [Student] = []
```
Comment (mvn-bienle-dn) [inline 03_SwiftOOP/ThongNT/BaiTap5-10.playground/Contents.swift:246]: Remove space

File: 03_SwiftOOP/ThongNT/BaiTap5-10.playground/Contents.swift [hunk 3]
```diff
@@ -0,0 +1,417 @@
...
+    case March
+    case April
+    case May
+    case June
+    case July
+    case August
+    case September
+    case October
+    case November
+    case December
```
Comment (mvn-bienle-dn) [inline 03_SwiftOOP/ThongNT/BaiTap5-10.playground/Contents.swift:294]: Start a case by lowercase character

File: 03_SwiftOOP/ThongNT/BaiTap5-10.playground/Contents.swift [hunk 4]
```diff
@@ -0,0 +1,417 @@
...
+
+class CustomDate {
+    var day: Int = 1
+    var month: Months
+    var year: Int = 1990
+    
+    init(day: Int, month: Months, year: Int) {
+        self.day = day
+        self.month = month
+        self.year = year
```
Comment (mvn-bienle-dn) [inline 03_SwiftOOP/ThongNT/BaiTap5-10.playground/Contents.swift:305]: Check "nam nhuan"

File: 03_SwiftOOP/ThongNT/BaiTap5-10.playground/Contents.swift [hunk 5]
```diff
@@ -0,0 +1,417 @@
...
+let date = CustomDate(day: 1, month: .December, year: 1999)
+date.moveDay(year: 1, month: 1, day: 32)
+print(date.toString())
+
+
+//-----------------------------------------//
+// Bài tập 9 //
+// Xây dựng mảng 1 chiều với các phần tử là số nguyên//
+final class Mang1c {
+    var arr : [Int] = []
```
Comment (mvn-bienle-dn) [inline 03_SwiftOOP/ThongNT/BaiTap5-10.playground/Contents.swift:367]: Remove space

File: 03_SwiftOOP/ThongNT/BaiTap5-10.playground/Contents.swift [hunk 6]
```diff
@@ -0,0 +1,417 @@
...
+    
+    func printArray() {
+        arr.forEach { print($0)}
```
Comment (mvn-bienle-dn) [inline 03_SwiftOOP/ThongNT/BaiTap5-10.playground/Contents.swift:370]: Add space before "}"

**Comment by mvn-thongnguyen-dn** [comment]
@mvn-bienle-dn Please check my fixes. Thank you.

**Review by mvn-bienle-dn — APPROVED** [review]
LGTM!

**Pull request merged**